test-app/frontend/src/Footer/Footer.scss

```diff
@@ -0,0 +1,40 @@
+@import "~bootstrap/scss/bootstrap";
+
+@font-face {
+    font-family: Sofia Pro Regular;
+    src: url("../../public/fonts/Sofia\ Pro\ Regular\ Az.otf");
+}
+
+@font-face {
+    font-family: Sofia Pro Regular;
+    src: url("../../public/fonts/Sofia\ Pro\ Bold\ Az.otf");
+    font-weight: bold;
+}
+
+:root {
+    /* variables */
+    --cg-black: rgb(30, 32, 32);
+    --cg-blue: rgb(6, 87, 152);
+    --cg-grey: rgb(243, 243, 243);
+    --cg-white: rgb(255, 255, 255);
+}
```

Comment:
SCSS actually allows us to use variables in a nicer way than CSS.

In `src`, you can make a `_global_variables.scss` file and put variables inside of it (example: https://github.com/bglossner/tree-tracking-app/blob/main/packages/frontend/src/_global-variables.scss).

Then in other files you can do `@use '../global-variables' as gv;` to import the variables and use them like `gv.$secondary-color;` instead of like `var(--<var>)`. This way, we can get autocomplete in editors and utilize some of the cool features of Sass. This will allow us to create global "themes" where we can define things as primary/secondary for coloring.